python/tvm/relay/tensorrt.py

```diff
@@ -738,7 +803,7 @@ def PruneSubgraphs(mod, compiler="tensorrt", use_implicit_batch=True, prune_no_m
         Which mode we plan to use for TensorRT. Will be used to determine which subgraphs are
         valid. In implicit batch mode, all inputs to a subgraph must have the same batch size.
 
-    prune_no_macs : bool
+    prune_compute_heavy_subgraphs : bool
```

Comment:
I don't think we need to change this name

Reply:
Reverted